src/index.html

```diff
@@ -8,20 +9,464 @@
     />
     <title>Catalog</title>
     <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
+      href="styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. Please ensure that the stylesheet is compiled to CSS and the correct path is used.